docs/09-membership.md

```diff
@@ -96,19 +96,19 @@ If someone is doing work you feel should be eligible but is not currently listed
  | EF Research | [Kevaundray Wedderburn](https://github.com/kevaundray/) | 1 |
  | EF Research | [Luca Zanolini](https://github.com/luca-zanolini) | 1 |
  | EF Research | [Pop Chunhapanya](https://github.com/ppopth/) | 1 |
```

Comment:
"EF Research" can be changed to "EF Consensus R&D" if we are being audited.

Reply:
added to the PR! thanks